components/director/internal/domain/scenarioassignment/converter.go

```diff
@@ -0,0 +1,53 @@
+package scenarioassignment
+
+import (
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/pkg/errors"
+)
+
+func NewConverter() *converter {
+	return &converter{}
+}
+
+type converter struct{}
+
+func (c *converter) FromInputGraphql(in graphql.AutomaticScenarioAssignmentSetInput, tenant string) (model.AutomaticScenarioAssignment, error) {
```

Comment:
To be consistent with other:
```suggestion
func (c *converter) FromInputGraphQL(in graphql.AutomaticScenarioAssignmentSetInput, tenant string) (model.AutomaticScenarioAssignment, error) {
```